refactor(napi/parser, linter/plugins): improve formatting of generated code

overlookmotel opened the pull request:
Tiny improvement to generated code. Break up the code with line breaks in more places, to make it a little easier to read.

camc314 reviewed (commented):
consider piping it through dprint?

overlookmotel committed:
refactor(napi/parser, linter/plugins): improve formatting of generated code (#14554)

Tiny improvement to generated code. Break up the code with line breaks in more places, to make it a little easier to read.
